com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/security/data/termination/DataTerminationService.java

```diff
@@ -120,13 +121,13 @@ protected CloudControllerClientImpl getCFClient() {
     private List<String> getDeleteSpaceEvents() {
         CloudControllerClientImpl cfClient = getCFClient();
         CFOptimizedEventGetter cfOptimizedEventGetter = new CFOptimizedEventGetter(cfClient);
-        return cfOptimizedEventGetter.findEvents(SPACE_DELETE_EVENT_TYPE, getDateBeforeTwoDays());
+        return cfOptimizedEventGetter.findEvents(SPACE_DELETE_EVENT_TYPE, getDateBeforeDays(GET_EVENTS_DAYS_BEFORE));
     }
 
-    private String getDateBeforeTwoDays() {

```

Comment:
The method name used to say 2 days, but the GET_EVENTS_DAYS_BEFORE was actually set to 1. Changed the name and made the method more generic to avoid similar inconsistencies again.